src/main/resources/application.properties

```diff
@@ -5,4 +5,16 @@ spring.http.encoding.enabled=true
 # Force the encoding to the configured charset on HTTP requests and responses
 spring.http.encoding.force=true
 # Disable weird Hibernate error logging
-logging.level.org.hibernate.engine.jdbc.internal.LobCreatorBuilder=ERROR
\ No newline at end of file
+logging.level.org.hibernate.engine.jdbc.internal.LobCreatorBuilder=NONE
+
+# Data configurations
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.username=postgres
+spring.datasource.password=password
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.jpa.properties.hibernate.dialect=com.monumental.CustomPostgreSQL9Dialect
+spring.jpa.properties.hibernate.connection.driver_class=org.postgresql.Driver
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+spring.jpa.show-sql = true
+spring.jpa.hibernate.ddl-auto = update
+spring.jpa.properties.hibernate.default_schema=public
```

Comment:
I *think* that this completely replaces hibernate.cfg.xml but I have not tested it fully